resources/views/auth/login.blade.php

```diff
@@ -40,7 +40,10 @@ class="flex flex-col px-4 py-8 mx-4 border-2 rounded-xl sm:px-8 border-theme-sec
                         $usernameAlt = Config::get('fortify.username_alt');
                         $type = 'text';
 
-                        if ($usernameAlt) {
+                        if ($username === $usernameAlt && ! Config::get('fortify.email') === null) {
```

Comment:
@Highjhacker Wouldn't `Config::get('fortify.email') !== null` make more sense?

Reply:
Or does this evaluate `! Config::get('fortify.email')` first, then compare to null? If that's the case, it would never pass as true/false will never be equal to null 🤔

Reply:
Just pushed now, think it should be better 🤔 Tried with every possible combination in the `fortify.php` config file, and got what I expected.

username & mail being used => "Username Or Mail"
username used, but not mail => "Username"
mail used, but not username => "Email"